src/components/Calendar/Calendar.cy.tsx

```diff
@@ -34,33 +34,33 @@ describe("<Calendar />", () => {
             year: "numeric",
         });
 
-        cy.mount(<Calendar initialEvents={[]} />);
+        cy.mount(<Calendar initialEvents={[]} initialDate={today} />);
         cy.get(".fc-toolbar-title").should("have.text", currentMonthYear);
     });
 
     it("events render", () => {
-        cy.mount(<Calendar initialEvents={sampleEvents} />);
+        cy.mount(<Calendar initialEvents={sampleEvents} initialDate={today} />);
         cy.get(".fc-event").should("be.visible");
     });
 
     it("can change view to timeGridDay", () => {
-        cy.mount(<Calendar initialEvents={[]} />);
+        cy.mount(<Calendar initialEvents={[]} initialDate={today} />);
         cy.get("button.fc-timeGridDay-button").click();
         cy.get(".fc-timeGridDay-view").should("be.visible");
     });
 
     it("can change view to timeGridWeek", () => {
-        cy.mount(<Calendar initialEvents={[]} />);
+        cy.mount(<Calendar initialEvents={[]} initialDate={today} />);
         cy.get("button.fc-timeGridWeek-button").click();
         cy.get(".fc-timeGridWeek-view").should("be.visible");
     });
 
     it("can change view between months in dayGridMonth", () => {
-        const prevMonth: Date = new Date();
+        const prevMonth: Date = new Date(today);
```

Comment:
can you get rid of `: Date` type annotation?
apologies, I know this was already on main and not a change you made, just now that I've noticed it it's going to annoy me!